cluster/up.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -e
+
+source ./cluster/gocli.sh
+
+num_nodes=${KUBEVIRT_NUM_NODES:-1}
+re='^-?[0-9]+$'
+if ! [[ $num_nodes =~ $re ]] || [[ $num_nodes -lt 1 ]] ; then
+    num_nodes=1
+fi
+
+image="k8s-1.10.4"
+case "${KUBEVIRT_PROVIDER}" in
```

Comment:
It looks like `KUBEVIRT_PROVIDER` is meant to be set at the cli - if so, then it would be nice to simplify the cases to `kubernetes` and `openshift` and set `k8s-1.10.4` or `os-3.10.0` accordingly.

e.g.
```
...
 "kubernetes")
    image = "${KUBE_IMAGE}"
...
```

Reply:
Given that, I'm fine with the values as is then.